application/comit_node/src/http_api/routes/rfc003/handlers/get_action.rs

```diff
@@ -101,15 +96,27 @@ pub enum ActionResponsePayload {
         amount: EtherQuantity,
         gas_limit: ethereum_support::U256,
         network: ethereum_support::Network,
+        min_block_timestamp: Option<Timestamp>,
     },
 }
 
-#[derive(Clone, Debug, Serialize)]
-pub struct ActionResponseBody {
-    #[serde(flatten)]
-    pub payload: ActionResponsePayload,
-    #[serde(skip_serializing_if = "Option::is_none")]
-    pub invalid_until: Option<Timestamp>,
+impl ActionResponseBody {
+    fn bitcoin_broadcast_signed_transaction(transaction: &Transaction, network: Network) -> Self {
+        let min_median_time = if transaction.lock_time == 0 {
+            None
+        } else {
+            // The first time a tx with lock_time can be broadcasted is when
+            // mediantime == locktime + 1
```

Comment:
```suggestion
            // mediantime > locktime
```
This is a weird way to phrase it.

Reply:
What is so weird about it?

The comment (both lines!) explains why we are doing `+ 1` to the `lock_time` and call it `min_median_time` afterwards.
I don't see how using different operations in the comments vs the actual code makes things easier to understand?

Reply:
It is weird because the consensus is that nLockTime must be strictly less than the median time for the transaction to be valid, not that it needs to be `+1`.
You translated that to `+1` in the code which mathematically correct but it is not what the consensus says, it is a consequence of the rule, not the rule.

Reply:
The consensus for info:
```
    if ((int64_t)tx.nLockTime < ((int64_t)tx.nLockTime < LOCKTIME_THRESHOLD ? (int64_t)nBlockHeight : nBlockTime))
        return true;
```